staging_notes.md

```diff
@@ -0,0 +1,153 @@
+is a background process and de-prioritized compared to running the network.
+
+#### replace.toml
+
+If you have modifications to config.toml that you wish to perform, a `replace.toml` file can be created to assist with this. 
+This file can be used during staging to automate the replace when a `config.toml` file is made from `config-example.toml` during staging.
+
+It should be a minimal version of `config.toml` with only files you wish to replace. And example to change `allow_request_get_all_values`
+is below:
+
+```toml
+[binary_port_server]
+allow_request_get_all_values = true
+```
+
+I prefer to keep `replace.toml` (filename is not important) in `/etc/casper/` with my other config files.
+
+To replace when staging protocol upgrades:
+```bash
+sudo -u casper /etc/casper/node_util.py stage_protocols casper.conf --replace /etc/casper/replace.toml 
+```
+
+If you have already staged and wish to change `config.toml` or just test the replacement, you can regenerate with:
+```bash
+sudo -u casper /etc/casper/node_util.py config_from_example 2_0_0 --replace /etc/casper/replace.toml
+```
+The output will indicate that since you have a `/etc/casper/2_0_0/config.toml`, `/ect/casper/2_0_0/config.toml.new` was created.
+
+### Testing config.toml
+
+If you make any changes to `config.toml` for the upcoming release, it is good to test them. If your node attempts to 
+automatically upgrade, but has a parsing error with the config file, it will refuse to start. We have a verification of
+parsing built into the node. This doesn't assure that values are correct, but only the node can parse the `config.toml`.
+
+To check this, you would use the same binary version as the release and a special run command:
+
+`/var/lib/casper/bin/2_0_0/casper-node validate-config /etc/casper/2_0_0/config.toml`
+
+Example output:
+
+```bash
+$/var/lib/casper/bin/2_0_0/casper-node validate-config /etc/casper/2_0_0/config.toml
+{"timestamp":"2025-04-28T13:30:42.694327Z","level":"INFO","fields":{"message":"config file is valid","build_version":"2.0.0-a7e4ff1","config_file":"/etc/casper/2_0_0/config.toml"},"target":"casper_node::cli"}
+{"timestamp":"2025-04-28T13:30:42.696457Z","level":"INFO","fields":{"message":"exiting casper-node","exit_code":"0"},"target":"casper_node"}
+```
+
+Example of known_addresses with missing quotes for an entry:
+
+```bash
+$ ./casper-node validate-config ../config/config-example.toml 
+2025-04-28T13:55:54.225122Z ERROR [casper_node::cli cli.rs:262] config file is not valid; build_version=2.0.0-a7e4ff1; config_file="/etc/casper/2_0_0/config.toml"
+Error: TOML parse error at line 194, column 61
+    |
+194 | known_addresses = ['3.225.137.28:35000','3.231.187.43:35000]
+    |                                                             ^
+invalid literal string
+```
+
+Example of a field I "accidentally" added to the config that doesn't belong:
```

Comment:
Rephrase to not be written in the first person

Reply:
Corrected